apps/judicial-system/web/src/routes/Prosecutor/Indictments/Indictment/Indictment.tsx

```diff
@@ -72,20 +74,83 @@ const Indictment: React.FC = () => {
 
   useDeb(workingCase, ['indictmentIntroduction', 'demands'])
 
+  const setDriversLicenseSuspensionRequest = useCallback(
+    (indictmentCounts?: TIndictmentCount[]) => {
+      let requestDriversLicenseSuspension = false
+
+      if (
```

Comment:
Nice! Just one more nitpick. What would you think about assigning this if statement directly to the `requestDriversLicenceSuspension` variable, i.e.

```
const requestDriversLicenceSuspension =  !indictmentCounts?.some((count) =>
    ...
```

That way we could skip this if statement and the `requestDriversLicenseSuspension = true` on line 104.